h2/events.py

```diff
@@ -136,6 +136,52 @@ def __repr__(self):
         )
 
 
+class _HeadersSent(object):
+    """
+    The _HeadersSent event is fired whenever headers are received.  This event
+    carries the HTTP headers and the stream ID of the new stream.
+
+    This is an internal event, used to determine validation steps on
+    outgoing header blocks.
+    """
+    def __init__(self):
+        #: The Stream ID for the stream this response was made on.
+        self.stream_id = None
+
+        #: The headers.
+        self.headers = None
```

Comment:
I just copied this field from some of the other internal events. But I can’t actually see where it gets populated on, say, `HeadersReceived`. Do we actually need it for these internal events, given that all we care about is whether this is a trailer or not?